jerry-core/ecma/operations/ecma-conversion.cpp

```diff
@@ -489,9 +489,6 @@ ecma_op_from_property_descriptor (const ecma_property_descriptor_t* src_prop_des
   else
   {
     // 4.
-    JERRY_ASSERT (src_prop_desc_p->is_get_defined
-                  || src_prop_desc_p->is_set_defined);
```

Comment:
@ruben-ayrapetyan I tried to set `undefined` value for the [[Get]] / [[Set]] field in the `ecma_get_property_descriptor_from_property` but I got another assert when I tried to create object from the `undefined` value (the `ecma_property_descriptor_t`'s  `get_p` and `set_p` are `ecma_object_t*`). Could you give any hint how this should be done?